extensions/interactions/GraphInput/validator.js

```diff
@@ -53,6 +56,46 @@ oppia.filter('oppiaInteractiveGraphInputValidator', [
     warningsList = warningsList.concat(
       baseInteractionValidationService.getAllOutcomeWarnings(
         answerGroups, defaultOutcome, stateName));
+    for (var i = 0; i < answerGroups.length; i++) {
+      var ruleSpecs = answerGroups[i].rule_specs;
+      for (var j = 0; j < ruleSpecs.length; j++) {
+        var ruleSpec = ruleSpecs[j];
+        try {
+          if (ruleSpec.rule_type === 'IsIsomorphicTo' &&
+              ruleSpec.inputs.g.vertices.length > ISOMORPHISM_VERTICES_LIMIT) {
+            exceedsIsomorphismVerticesLimit = true;
+          }
+
+          if (ruleSpec.inputs.g.vertices.length > VERTICES_LIMIT) {
+            exceedsVerticesLimit = true;
+          }
+        } catch (e) {
+          warningsList.push({
+            type: WARNING_TYPES.CRITICAL,
+            message: (
+              'The graph used in the rule ' + String(j + 1) +
```

Comment:
Um, I don't think you addressed my concern here. I think this message was fine as it was (i.e., "something went wrong"); I was referring to the ones below, which don't include any information about the answer group or the rule that is problematic.

Basically, imagine if you had 10 answer groups and the second one had a graph that had too many vertices. When you look at the warnings on the exploration overview, how would you tell that it was the second answer group that had a problem? That's what I was pointing out.